web_app/tests/test_PositionDBConnector.py

```diff
@@ -0,0 +1,235 @@
+def sample_position(sample_user):
+    """Fixture to create a sample position for testing."""
+    return Position(
+        user_id=sample_user.id,
+        token_symbol="ETH",
+        amount="100",
+        multiplier=2,
+        status=Status.PENDING.value,
+        start_price=0.0
+    )
+
+
+@pytest.fixture(scope="function")
+def mock_session():
+    """Fixture to create a mock database session."""
+    session = MagicMock()
+    return session
+
+
+
+### Positive Test Cases ###
+
+def test_position_to_dict(mock_position_db_connector, sample_position):
+    """Test converting a Position object to dictionary."""
+    result = PositionDBConnector._position_to_dict(sample_position)
+    
+    assert isinstance(result, dict)
+    assert result["id"] == str(sample_position.id)
+    assert result["user_id"] == str(sample_position.user_id)
+    assert result["token_symbol"] == sample_position.token_symbol
+    assert result["amount"] == sample_position.amount
+    assert result["multiplier"] == sample_position.multiplier
+    assert result["status"] == sample_position.status
+
+
+def test_get_positions_by_wallet_id_success(mock_position_db_connector, sample_user, sample_position):
+    """Test successfully retrieving positions by wallet ID."""
+    mock_position_db_connector._get_user_by_wallet_id.return_value = sample_user
+    
+    position_dict = {
+        "id": str(sample_position.id),
+        "user_id": str(sample_position.user_id),
+        "token_symbol": sample_position.token_symbol,
+        "amount": sample_position.amount,
+        "multiplier": sample_position.multiplier,
+        "status": sample_position.status,
+        "start_price": sample_position.start_price
+    }
+    
+    mock_position_db_connector.get_positions_by_wallet_id.return_value = [position_dict]
+
+    positions = mock_position_db_connector.get_positions_by_wallet_id("test_wallet_id")
+
+    assert len(positions) == 1
+    assert positions[0]["id"] == str(sample_position.id)
+    assert positions[0]["token_symbol"] == sample_position.token_symbol
+    assert positions[0]["amount"] == sample_position.amount
+    assert positions[0]["multiplier"] == sample_position.multiplier
+    assert positions[0]["status"] == sample_position.status
+
+
+def test_create_position_success(mock_position_db_connector, sample_user):
+    """Test successfully creating a new position."""
+    mock_position_db_connector._get_user_by_wallet_id.return_value = sample_user
+    
+    new_position = Position(
+        user_id=sample_user.id,
+        token_symbol="BTC",
+        amount="200",
+        multiplier=3,
+        status=Status.PENDING.value,
+        start_price=0.0
+    )
+    mock_position_db_connector.write_to_db.return_value = new_position
+    mock_position_db_connector.create_position.return_value = new_position
+    
+    result = mock_position_db_connector.create_position(
+        wallet_id="test_wallet_id",
+        token_symbol="BTC",
```

Comment:
I don't think that we use `BTC` anywhere in the code, please use some values from TokenParams.ETH.name